### PR TITLE
Support imagestreams RHEL9

### PR DESCRIPTION
**Pull request by phracek**
Support testing image stream for RHEL9 and testing RHEL9 and 10.5 for template

### CODE REVIEW

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
RHEL9 test for imagestreams passed:

```bash
-----------------------------------------------
Running test test_mariadb_imagestream (starting at 2022-10-14 14:05:57+02:00) ... 
-----------------------------------------------
Running image stream test for stream test/imagestreams/mariadb-rhel.json and template test/examples/mariadb-ephemeral-template.json
Now using project "sclorg-test-16748" on server "https://api.core-serv-ocp.hosted.psi.rdu2.redhat.com:6443".

You can add applications to this project with the 'new-app' command. For example, try:

    oc new-app rails-postgresql-example

to build a new example application in Ruby. Or use kubectl to deploy a simple Kubernetes application:

    kubectl create deployment hello-node --image=k8s.gcr.io/e2e-test-images/agnhost:2.33 -- /agnhost serve-hostname

secret/6340056-core-services-apps-ci-pull-secret created
imagestream.image.openshift.io/mariadb created
secret/mariadb created
service/mariadb created
deploymentconfig.apps.openshift.io/mariadb created
Waiting for mariadb pod becoming ready ........... DONE
project.project.openshift.io "sclorg-test-16748" deleted
Project sclorg-test-16748 was deleted properly
```

File: test/test-lib-mysql.sh [hunk 1]
```diff
@@ -129,18 +129,24 @@ function test_mariadb_integration() {
...
+  elif [ "${OS}" == "rhel9" ]; then
+    tag="-el9"
   fi
   ct_os_test_image_stream_template "${THISDIR}/imagestreams/mariadb-${OS%[0-9]*}.json" "${THISDIR}/examples/mariadb-ephemeral-template.json" mariadb "-p MARIADB_VERSION=${VERSION}${tag}"
 }
 
 function test_mariadb_template() {
-  if [[ "${OS}" =~ rhel7 ]] || [[ "${OS}" =~ centos7 ]] || [[ "${OS}" =~ rhel8 ]] ; then
+  if [[ "${OS}" =~ rhel9 ]] && [[ "${VERSION}" == "10.3" ]]; then
+    echo "Testing mariadb_template is not supported for RHEL9 and version 10.3"
```
Comment (zmiklank) [inline test/test-lib-mysql.sh:146]: Why?

Comment (phracek) [reply test/test-lib-mysql.sh:146]: mariadb-10.3 on RHEL9 is not supported.

Comment (zmiklank) [reply test/test-lib-mysql.sh:146]: We should change this condition as discussed in person.

Comment (phracek) [reply test/test-lib-mysql.sh]: resolved by https://github.com/sclorg/mariadb-container/pull/190/commits/a27dcae273c18f134e223b17ebc45e15a798a93d

**Comment by phracek** [comment]
[test-all]

**Review by zmiklank — APPROVED** [review]

**Pull request merged**